### ISSUE TITLE
Link with https://github.com/tendermint/atom_one 

### ISSUE THREAD

**Issue by moul** [atomone-hub/genesis#22]
What’s the current and expected upcoming relationship between this repo and https://github.com/tendermint/atom_one?

**Comment by jaekwon** [comment]
Yeah let's move everything from tendermint/atom-one that is relevant, to this repo. 
Also I think the Decentralists DAO governance suggestions, should be incorporated too either in the constitution or a separate file for governance changes.
